constans/scenarios.py

```diff
@@ -556,31 +556,103 @@ def board_player_1_scenario():
 TEST_PLAYERS_CHARACTER_2 = Player(PLAYER_1)
 TEST_PLAYERS_CHARACTER_2.score = 1000
 
+
 # shoot scenarios
-SCENARIOS_SHOOT_TEST = [[Cell(i, j) for j in range(LARGE)] for i in range(LARGE)]
-CELL_SHOOT_8_8 = Cell(8, 8)
-SHOOTER_PLAYER = Player(PLAYER_1)
-SHOOTED_PLAYER = Player(PLAYER_2)
-CELL_SHOOT_8_8.character = SHOOTER_PLAYER.characters[0]
-CELL_SHOOT_8_8.is_discover[0] = True
+def generate_board_for_shoot_action_test():
+    scenario = [[Cell(i, j) for j in range(LARGE)] for i in range(LARGE)]
+
+    shooter_player = Player(PLAYER_1)
+    shooted_player = Player(PLAYER_2)
+
+    # add one more character to test cases
+    shooter_player.characters.append(Character(shooter_player))
+    shooted_player.characters.append(Character(shooted_player))
```

Comment:
Past tense of `shoot` is `shot`. You will have to change this across the whole PR